components/email-form/EmailFormContainer.tsx

```diff
@@ -0,0 +1,80 @@
+import { Button, Stack, Typography, useTheme } from '@mui/material'
+import { useState } from 'react'
+
+import EmailFormInput from './EmailFormInput'
+
+const EmailFormContainer = () => {
+  const theme = useTheme()
+  const [email, setEmail] = useState('');
+  const [error, setError] = useState('');
+
+  function handleSubmit(e) {
+    e.preventDefault();
+    const isValidEmail = /^[\w-\.]+@([\w-]+\.)+[\w-]{2,4}$/g;
```

Comment:
@mistyb01 I would like to discuss tomorrow regex for email. Thank you!

Reply:
It would be better to push down some of the details into a utils or newsletter module.  easier testing and reuse.
We can address that when we make a feature flag.

Reply:
@jnakaso agree with you.